### DISCUSSION TITLE
Attribution text with links

### DISCUSSION THREAD

**Discussion by buddhaCode** [swisnl/filament-backgrounds#15] [Ideas]
Hey @JaZo,
I was wondering if it's possible to use links `<a>` in the attribution text. Looking at the source code, I found out, that this plugin uses the CSS `content` property to set the text. This implementation doesn't allow HTML tags. Do you plan on changing the implementation to something that would allow HTML tags?

Best,
Arne

**Comment by JaZo** [comment]
Correct, the current implementation doesn't support links in the attribution. I tried to keep this plugin as unobtrusive as possible so that it would be compatible with existing themes. However, since it's nice to backlink to the photographers, I'm open to a PR that would add support.

**Reply by buddhaCode** [reply]
Yeah, it's always good to be as unobtrusive as possible.

I'll have a look at it and try to make a PR. But it may take some days or weeks until I'll get to it.
